### PR TITLE
made guide on how to make new projects

### PR DESCRIPTION
**Pull request by andreavendrik**

### CODE REVIEW

File: new-projects/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# How to make a new project
+
+So you have an amazing idea that would benefit the client, or New Atoms to the moon and back? Great. Let's make a new project.
+
+### Get started
+To prevent our workflow from slowing down, there is no need to vote in achievables for making new projects.
+Instead, you can make a new project on your own accord.
+
+To do so, make a new folder in the projects folder of the relevant repository. Keep the folder name as short possible, yet descriptive. So rather than 'how-to-make-a-new-project', call it 'new-projects'. To this folder you add a) title.txt for the title of your project and b) readme.md, in which you put the description of you project.
+
```
Comment (SPrinss) [inline new-projects/readme.md:12]: is readme in addition to pitch.md?

The interface will only display title.txt, pitch.md and post.md (and social posts).

File: new-projects/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# How to make a new project
+
+So you have an amazing idea that would benefit the client, or New Atoms to the moon and back? Great. Let's make a new project.
+
+### Get started
+To prevent our workflow from slowing down, there is no need to vote in achievables for making new projects.
+Instead, you can make a new project on your own accord.
+
+To do so, make a new folder in the projects folder of the relevant repository. Keep the folder name as short possible, yet descriptive. So rather than 'how-to-make-a-new-project', call it 'new-projects'. To this folder you add a) title.txt for the title of your project and b) readme.md, in which you put the description of you project.
```
Comment (SPrinss) [inline new-projects/readme.md]: Repository, achievables, project can get hyperlinked

File: new-projects/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### Fill in the blanks (and give them proper thought)
+A new project always has a Goal, [Definition of Done](define-done), Proposed Methods and Motivation. Take some time to consider these carefully. The clearer you describe your project, the better the results will be.
+
```
Comment (SPrinss) [inline new-projects/readme.md:15]: define-done
I think it does need the `.md` behind it

File: new-projects/readme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+### Get it reviewed
+Finished? Awesome! To get it reviewed, make a new achievable. This should be easy – the Goal of the achievable can mirror the goal of your project, and the proposed I Can is simply the creation of your new project.
+Link it to your pull request, and put it in the Done straightaway!
+
```
Comment (SPrinss) [inline new-projects/readme.md:19]: `pull request` and `done` might also be linked.

File: new-projects/readme.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+### Get your project commissioned
+Once merged, the reviewer will add your new project to the Inbox of the concerned product. Only once it gets commissioned by the product owner you can start adding achievables.
```
Comment (SPrinss) [inline new-projects/readme.md]: We vote them in as a team no? Or did that change? :)

**Comment by andreavendrik** [comment]
Great comments @SPrinss ! updated

**Pull request merged**